syntax = "proto3";

package index;

import "google/protobuf/empty.proto";

message IndexResponse {
  string project_name = 1;
  string app_name = 2;
  string app_version = 3;
}

service IndexService {
  rpc GetInfo(google.protobuf.Empty) returns (IndexResponse) {}
}
